tst/working/quick/Sporadic.tst

```diff
@@ -44,19 +44,13 @@ gap> TestSporadic("Suz");
 [ "Suz" ]
 gap> TestSporadic("Fi23");
 [ "Fi23" ]
-gap> TestSporadic("M12.2");
-[ "M12.2" ]
```

Comment:
So it seems these extensions are not dealt with by the new code. How does Magma deal with them, then?

Reply:
We're now discussing this in #196.